doc/NotaryContentTrust.md

```diff
@@ -79,33 +94,35 @@ Notary needs authorization credentials to communicate to the Notary Server. For
 
 Note: Replace username and password with actual Service Principal credentials. 
 
-#### Notary TUF Trust collection initialization
+##### c. Notary TUF Trust collection initialization
 Once the Notary is set up, the TUF trust collection for each individual image can be done.
 
 `notary init --rootkey root_image.key --rootcert root_image.crt exampleregistry.azurecr.io/image -c ~/.notary/config.json`
 
 Note that Notary Trust collection for an container image can be initialized only once. If a new version of the image there is no need to initialize the trust collection again. 
 
-#### Notary Snapshot key rotation 
+##### d. Notary Snapshot key rotation 
 
 TUF Snapshot key must be rotated to the Notary server. With this command, there is no need to maintain the private key of the Snapshot key locally at Publisher instead it will be maintained by the Notary Server. 
 
 `notary key rotate exampleregistry.azurecr.io/image snapshot -r -c ~/.notary/config.json`
 
-### 3. Sign and publish images using Docker Client
+## Step 3. Sign and publish images using Docker Client
 
 Once the trust collection is created, the container image can be signed with Docker client. First login into the Container Registry by `docker login exampleregistry.azurecr.io`. Provide the Service Principal username and password to sign in. 
 
+##### a.  Docker Environment variable configuration
 To enable Docker Content Trust, set the environment variable as `export DOCKER_CONTENT_TRUST=1`
 
+##### b. Sign and Publish
 To sign the target container image, `docker push exampleregistry.azurecr.io/image:v1`. Tag must be specified while signing the image. Step 3 has to be repeated whenever there is new update in the image. Now the image is successfully signed and pushed into the registry.
 Note: Target role private key pass code is same as the repository key pass code. 
```

Comment:
Question - what does this mean if one uses Service Principal instead of passcode to access the repository?

Reply:
the Service Principal also has a username and password similar to the passcode way to access the repository